build.sh

```diff
@@ -12,18 +12,21 @@ shift "$(( OPTIND - 1 ))"
 
 NAME=$(echo "${PWD##*/}" | tr _ -)
 TAG="latest"
+MULTISTAGE_TARGET="development"
 
 UID="$(id -u "${USER}")"
 GID="$(id -g "${USER}")"
 
 if [ "$REBUILD" -eq 1 ]; then
     docker build \
         --no-cache \
+        --target "${MULTISTAGE_TARGET}" \
```

Comment:
I would propose it like that, we don't know yet how we're going to use it but if we put it there, its at least clear that this option exists.